### PR TITLE
10159 Disable PWA

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* /bcgov/entity#10159

_This is a bit experimental because it's not testable locally (PWA is not used locally). However, the cached HEAD call reproduces for me in Dev, so I will be able to verify this fix after it's merged._

*Description of changes:*
- added plugin to remove existing service workers on users' systems
- removed plugin to implement Vue PWA
- updated Vue config
- deleted obsolete manifest file
- deleted obsolete icons
- removed service worker registration
- deleted obsolete service worker scripts
- app version: 4.2.0

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -41,7 +41,8 @@
     "vue2-filters": "^0.10.0",
     "vuelidate": "^0.7.5",
     "vuetify": "^2.3.9",
-    "vuex": "^3.5.1"
+    "vuex": "^3.5.1",
+    "webpack-remove-serviceworker-plugin": "^1.0.0"
```
Comment (severinbeauvais) [inline package.json:45]: You can read up on this in the references I put in vue.config.js.

TLDR; You need to register a new Service Worker to replace the one currently on users' machines. But this new SW self-destroys. After some time (a year?) we can delete this package, since all users would have "wiped" their computers by then.

File: vue.config.js [hunk 2]
```diff
@@ -11,6 +12,10 @@ const aboutText2 = (sbcName && sbcVersion) ? `${sbcName} v${sbcVersion}` : ''
 module.exports = {
   configureWebpack: {
     plugins: [
+      // this is needed to remove existing service workers on users' systems
+      // ref: https://www.npmjs.com/package/webpack-remove-serviceworker-plugin
+      // ref: https://github.com/NekR/self-destroying-sw/tree/master/packages/webpack-remove-serviceworker-plugin
+      new RemoveServiceWorkerPlugin(),
```
Comment (severinbeauvais) [inline vue.config.js:18]: No filename is specified here because the default one is OK. After a `npm run build`, the dist/ folder includes file `sw.js` that is installed by the plugin.

File: package.json (hunk 1 above)
Comment (severinbeauvais) [reply package.json:45]: New ticket to remove this plugin in 1 year: https://app.zenhub.com/workspaces/entities-team-space-6143567664fb320019b81f39/issues/bcgov/entity/10370

**Review by lekshmimallika-aot — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (cameron-freshworks) [reply package.json:45]: cool!

**Review by cameron-freshworks — APPROVED** [review]
Nothing better than a PR that REMOVES code 😁
